worker/src/core/loaders/message-worker/govsg.class.ts

```diff
@@ -141,36 +142,8 @@ class Govsg {
           apiClientIdMap.get(result.recipient) ?? WhatsAppApiClient.clientTwo,
         language: WhatsAppLanguages.english,
       }))
-      const validatedWhatsAppIds =
-        await this.whatsappClient.validateMultipleRecipients(
-          unvalidatedMessages,
-          config.get('env') === 'development'
-        )
-      const invalidMessages = validatedWhatsAppIds.filter(
-        (message) => message.status === 'failed'
-      )
-      // update govsg_ops table with invalid messages
-      if (invalidMessages.length > 0) {
-        await this.postmanConnection.query(
-          `UPDATE
-              govsg_ops
-            SET
-              status = 'INVALID_RECIPIENT',
-              updated_at=clock_timestamp()
-            WHERE
-              id IN (:ids);`,
-          {
-            replacements: { ids: invalidMessages.map((message) => message.id) },
-            type: QueryTypes.UPDATE,
-          }
-        )
-      }
-
-      const validMessages = validatedWhatsAppIds.filter(
-        (message) => message.status === 'valid'
-      )
 
-      if (validMessages.length === 0) {
+      if (messagesToSend.length === 0) {
```

Comment:
If no more of whatsapp number validation then we don't need to keep this if condition already as nothing will be filtered

Reply:
good point